serde_someip/src/de.rs

```diff
@@ -601,27 +545,64 @@ impl<'de, 'a> SeqAccess<'de> for CommaSeparated<'a, 'de> {
         T: DeserializeSeed<'de>,
     {
         // Check if there are no more elements.
-        _seed.deserialize(&mut *self.de).map(Some)
+        _seed.deserialize(&mut *self.de).map(Some) //todo!()
+    }
+}
+
+struct CommaSeparatedVec<'a, 'de: 'a> {
```

Comment:
naming of this type is confusing.  SOME/IP doesn't have a concept of "Comma separation".   I think the intent of this type is to iterate over a SOME/IP vector with variable number of elements.  Perhaps an appropriate name for this is SomeIpVec